src/upgrade.rs

```diff
@@ -116,8 +120,8 @@ fn upgrade_exe(latest: &LatestLal, exe: &ExeInfo) -> LalResult<()> {
     http_download_to_path(&latest.url, &tar_dest)?;
     info!("Backing up {} to {}", exe.path, old_file.display());
     fs::rename(&exe.path, &old_file)?; // need to undo this if we fail
-    // NB: DO NOT INSERT CALLS THAT CAN FAIL HERE BEFORE THE OVERWRITE
-    // 3. force dump lal tarball into exe.prefix - rollback if it failed
+                                       // NB: DO NOT INSERT CALLS THAT CAN FAIL HERE BEFORE THE OVERWRITE
+                                       // 3. force dump lal tarball into exe.prefix - rollback if it failed
```

Comment:
go home rustfmt, you're drunk

Reply:
I added an extra line here, seems to have made it a bit better.